<?xml version="1.0" ?>
<testsuite errors="0" failures="2" name="pinktest.medianfilter.medianfilter" tests="2" time="0.000">
	<testcase classname="pinktest.medianfilter.medianfilter" name="test_0" time="0.000">
		<failure message="False is not true" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "/home/ujoimro/doc/projects/pink/pink-svn/python/pinktest/medianfilter.py", line 28, in test_0
    self.assertTrue( result == gold )
AssertionError: False is not true
]]>		</failure>
	</testcase>
	<testcase classname="pinktest.medianfilter.medianfilter" name="test_1" time="0.000">
		<failure message="False is not true" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "/home/ujoimro/doc/projects/pink/pink-svn/python/pinktest/medianfilter.py", line 34, in test_1
    self.assertTrue( result == gold )
AssertionError: False is not true
]]>		</failure>
	</testcase>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
